index.js

```diff
@@ -129,7 +193,20 @@ function filterByGenre() {}
       }
     ];
  */
-function getAllMoviesReleasedAtOrBeforeYear() {}
+function getAllMoviesReleasedAtOrBeforeYear(movies, year) {
+  let movieReleaseYear = [];
+
+  for(let movie of movies){
+    let releaseDate = movie.released.split(' ');
+    let yearNum = Number(releaseDate[2])
```

Comment:
We could do this in one line... or at least in one variable here by just reassigning.